.gitignore

```
@@ -20,6 +20,7 @@
 .env.production.local
 /**/.idea
 /**/.vscode
+/**/recordedData
```

```suggestion
/recordedData
```
We don't want to ignore all `recordedData` directories, just the top-level one. We do want to exclude `/Backend/recordedData/processedData/*.csv` and `/Backend/recordedData/sessions/*.bin` though